console/package.json

```diff
@@ -40,16 +40,16 @@
 		"classnames": "^2.3.2",
 		"ilib": "^14.15.1",
 		"immer": "^9.0.19",
-		"mapbox-gl": "^0.54.1",
+		"mapbox-gl": "^2.13.0",
```

Comment:
did you check the release information for this dependency?
I see this breaking change: https://github.com/mapbox/mapbox-gl-js/releases?page=7

[v2.0.0](https://github.com/mapbox/mapbox-gl-js/releases/tag/v2.0.0)
⚠️ Breaking changes
mapbox-gl-js is no longer under the 3-Clause BSD license. By upgrading to this release, you are agreeing to [Mapbox terms of service](https://www.mapbox.com/legal/tos/). Refer to LICENSE.txt for the new licensing terms and details. For questions, contact our team at https://support.mapbox.com/.



I think we should keep version 1.x as a dependecy to keep the open source use

Reply:
I have decided to roll back to version ^1.13.0